templates/server/puppet.conf.main.erb

```diff
@@ -0,0 +1,13 @@
+## Server config
+
+    # Puppet 3.0.x requires this in both [main] and [master] - harmless on agents
+    autosign       = <%= scope.lookupvar('::puppet::autosign') %>
```

Comment:
Sorry for the lag, missed that notification. I'm likewise fairly certain that this is no longer required, but I haven't tested it either.